pkg/webui/console/components/gateway-data-form/index.js

```diff
@@ -134,16 +134,17 @@ class GatewayDataForm extends React.Component {
 GatewayDataForm.propTypes = {
   children: PropTypes.node.isRequired,
   error: PropTypes.error,
-  formRef: PropTypes.object,
-  initialValues: PropTypes.object,
-  mapErrorsToFields: PropTypes.object,
+  formRef: PropTypes.shape({}),
```

Comment:
```suggestion
  formRef: PropTypes.shape({ current: PropTypes.instanceOf(Element) })
```

Reply:
this results in ```Warning: Failed prop type: Invalid prop `formRef.current` of type `Formik` supplied to `GatewayDataForm`, expected instance of `Element`.```

Reply:
Then `formRef: PropTypes.shape({ current: PropTypes.instanceOf(Formik) })` ?